xcube/core/resample.py

```diff
@@ -23,32 +23,43 @@
 
 import xarray as xr
 
+from xcube.core.schema import CubeSchema
 from xcube.core.select import select_vars
+from xcube.core.verify import assert_cube
 
 
 def resample_in_time(cube: xr.Dataset,
                      frequency: str,
                      method: Union[str, Sequence[str]],
                      offset=None,
-                     base: str = 0,
+                     base: int = 0,
                      tolerance=None,
                      interp_kind=None,
+                     time_chunk_size=None,
                      var_names: Sequence[str] = None,
-                     metadata: Dict[str, Any] = None):
+                     metadata: Dict[str, Any] = None,
+                     cube_asserted: bool = False) -> xr.Dataset:
     """
     Resample a xcube dataset in the time dimension.
 
     :param cube: The xcube dataset.
     :param frequency: Resampling frequency.
     :param method: Resampling method or sequence of resampling methods.
-    :param offset: Offset used to adjust the resampled time labels. Some pandas date offset strings are supported.
-    :param base: Resampling method.
+    :param offset: Offset used to adjust the resampled time labels.
+        Some pandas date offset strings are supported.
```

Comment:
Would it be possible to add a link to a doc that lists valid offset strings are supported?

Reply:
Its better saying that it uses same syntax as `frequency`.  Done.